packages/paste-website/src/pages/components/tabs/index.mdx

```diff
@@ -224,76 +284,45 @@ const HorizontalTabsExample: React.FC = () => {
-
-Changes the Tabs' direction to either have the panels below or on the right side.
-
-##### `variant?: 'fitted'| 'inverse' | 'inverse_fitted' | undefined (default)`
-
-Changes the Tabs' to either fit the width of the containing element or not.
-
-##### `selectedId?: string | null`
-
-Tells the Tabs which panel to load when it mounts. Important to provide, otherwise the first render may be a little delayed.
-
-##### `baseId?: string`
-
-Sets the prefix for the auto-generated ids on each Tab and TabPanel. Useful for consistent renders, like for tests.
-
-###### `element?: string`
-
-Overrides the default element name ('HORIZONTAL_TABS') to apply unique styles with the Customization Provider. If the Tabs are vertical, the default element value is 'VERTICAL_TABS'.
-
-#### TabList Props
-
-##### `aria-label: string`
-
-Required label for this Tabs component. Titles the entire Tabbing context for screen readers.
-
-###### `element?: string`
-
-Overrides the default element name ('HORIZONTAL_TAB_LIST') to apply unique styles with the Customization Provider. If the Tabs are vertical, the default element value is 'VERTICAL_TAB_LIST'.
-
-#### Tab Props
-
-##### `id?: string | undefined`
-
-Same as the HTML attribute.
-
-##### `focusable?: boolean | undefined`
-
-When an element is disabled, it may still be focusable.
-It works similarly to readOnly on form elements. In this case, only aria-disabled will be set.
-
-##### `disabled?: boolean | undefined`
-
-Same as the HTML attribute.
-
-###### `element?: string`
-
-Overrides the default element name ('HORIZONTAL_TAB') to apply unique styles with the Customization Provider. If the Tabs are vertical, the default element value is 'VERTICAL_TAB'.
-
-#### TabPanels Props
+#### Tabs props
 
-###### `element?: string`
+| Property     | Variants                                           | Description                                                                                                                  | Default                              |
+| ------------ | -------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- | ------------------------------------ |
+| orientation? | "horizontal", "vertical", "undefined"              | Changes the Tabs' direction to either have the panels below or on the right side.                                            | "horizontal"                         |
+| variant?     | "fitted", "inverse", "inverse_fitted", "undefined" | Changes each Tab to either equally fit the width of the containing element or hug the contents of its label.                 | "undefined"                          |
+| selectedId?  | `string, null`                                     | Tells the Tabs which panel to load when it mounts. Important to provide, otherwise the first render may be a little delayed. | `null`                               |
```

Comment:
todo: we don't normally put `, null` on these when the name has a `?`. Also `or` is usually signified by `|`

Reply:
is there a way to escape markdown to use the `|` symbol? I can't find any examples on the docs site of using that for `or` in a table

Reply:
`\|` like this

Reply:
update from slack: will consider using `|` to signify `or` in tables going forward when docs templates are standardized. right now it's more common practice for us to use `,` in props tables